ironfish/src/blockchain/blockchain.ts

```diff
@@ -1313,10 +1291,9 @@ export class Blockchain {
     Assert.isNotNull(prev.noteSize)
     Assert.isNotNull(prev.nullifierSize)
 
-    await Promise.all([
-      this.notes.truncate(prev.noteSize, tx),
-      this.nullifiers.truncate(prev.nullifierSize, tx),
-    ])
+    await this.notes.truncate(prev.noteSize, tx)
+
+    await this.nullifiers.disconnectBlock(block.transactions, tx)
```

Comment:
These can't be done in parallel anymore?

Reply:
This function is called disconnetBlock() but doesn't take a block. We should rename it right?

Reply:
Updated to take a block